client/verta/verta/_internal_utils/_artifact_utils.py

```diff
@@ -30,6 +30,12 @@
 }
 
 
+KERAS_H5PY_ERROR = RuntimeError(  # https://github.com/h5py/h5py/issues/1732
+    "Keras encountered an error saving/loading the model due to a bug in h5py v3.0.0;"
+    " consider downgrading with `pip install \"h5py!=3.0.0\"`"
```

Comment:
This is not necessarily a downgrade. Are we *sure* that 3.0.1 or whatever comes after has a fix?

Reply:
>This is not necessarily a downgrade.

Good point!

>Are we sure that 3.0.1 or whatever comes after has a fix?

"Sure" insofar as the h5py team have it [tracked as a bug](https://github.com/h5py/h5py/issues/1732) and are [preparing a fix](https://github.com/h5py/h5py/pull/1734) for their coming release.

@conradoverta Would it make more sense to just suggest `pip install "h5py<3.0.0"`, and _maybe_ update the message later with `!=` if the bug is fixed?

Reply:
Yep.